dispatch/static/manager/src/js/components/ZoneEditor/WidgetField.js

```diff
@@ -7,14 +7,29 @@ import { FormInput } from '../inputs'
 export default function WidgetField(props) {
   const Field = fields[props.field.type]
 
+  let error = ''
+  let errors = null
```

Comment:
intentional. 
one is for the widget field itself and the other the array of children field's errors.
I'll rename to clarify.